connectors/src/connectors/google_drive/temporal/utils.ts

```diff
@@ -13,6 +13,10 @@ export function getDocumentId(driveFileId: string): string {
   return `gdrive-${driveFileId}`;
 }
 
+export function getDriveId(documentId: string): string {
+  return documentId.replace(/^gdrive-/, "");
```

Comment:
wait, what?

Reply:
as discussed IRL: since all internal IDs are now prefixed, we need to remove the prefixes sometimes for db queries on a column that contains Google IDs (w/o prefixes), this is unsatisfying and increase entropy but we will leave that for after ECS indexation